src/components/Dropdown/index.jsx

```
@@ -28,75 +29,78 @@ const Dropdown = ({
-                  value: newValue.id,
-                },
-              };
-              onValueChange(syntheticEvent);
-            }
+            const syntheticEvent = {
+              target: { value: newValue ? newValue.id : '' },
+            };
+            onValueChange(syntheticEvent);
           }}
-          getOptionLabel={option => option.name || ''}
+          getOptionLabel={option => option.name || (typeof option.option === 'string' ? option.option : '') || ''}
```

ESLint

[error] Replace `·option.name·||·(typeof·option.option·===·'string'·?·option.option·:·'')·||·''` with `⏎············option.name·||⏎············(typeof·option.option·===·'string'·?·option.option·:·'')·||⏎············''⏎··········`

(prettier/prettier)